src/reducers/related-views.ts

```diff
@@ -0,0 +1,17 @@
+import { Action } from '../actions/index';
+import { RELATED_VIEWS_HIDE_TOGGLE } from '../actions/related-views';
+import { DEFAULT_RELATED_VIEWS, RelatedViews } from '../models/related-views';
+
+export function relatedViewsReducer(relatedViewToggler: RelatedViews = DEFAULT_RELATED_VIEWS,
+                                    action: Action): RelatedViews {
```

Comment:
FYI, I actually don't like when a code is tabbed by half a page.  Perhaps better to tab by two or do

```
export function relatedViewsReducer(
  relatedViewToggler: RelatedViews = DEFAULT_RELATED_VIEWS,
  action: Action
): RelatedViews {
```